src/Monitoring/OperationTimer.php

```diff
@@ -58,7 +58,7 @@ public function timeOperation(
         ?string $default_metric_name = NULL,
         ?string $default_source = NULL
     ) {
-        $metric     = new MetricId($default_metric_name, $default_source);
+        $metric     = MetricId::nameAndSource($default_metric_name, $default_source);
```

Comment:
Suppose this is the only downside to the named constructor / magic const - means here you'd have to pass the `::SOURCE_HOST_REPLACEMENT` into the $default_source arg.

I can't think of a better way to do that though at the moment, probably should make a note for the next major to break BC and have the method take a `MetricID` instead of the two strings....?